docs/code_examples/parallel_rps.py

```diff
@@ -161,9 +164,11 @@ def step(self, actions):
         env_truncation = self.num_moves >= NUM_ITERS
         truncations = {agent: env_truncation for agent in self.agents}
 
-        # current observation is just the other player's most recent action
+        # Current observation is just the other player's most recent action
+        # This is converted to a numpy value of type int to match the type
+        # that we declared in observation_space()
         observations = {
-            self.agents[i]: int(actions[self.agents[1 - i]])
+            self.agents[i]: np.array(actions[self.agents[1 - i]], dtype=np.int_)
```

Comment:
Is np.int_ the correct dtype?

Reply:
Actually, it looks like np.int64 may be the most correct choice. int_ is the same on most platforms, but maybe not all

Reply:
If you could fix this I can merge it

Reply:
I pushed a fix the pr